src/mission-planner/cfg/mission_planner.cfg

```diff
@@ -0,0 +1,17 @@
+#test config file for the mission planner
+
+PACKAGE='mission-planner'
+import roslib; roslib.load_manifest(PACKAGE)
+
+from dynamic_reconfigure.parameter_generator import *
+
+gen = ParameterGenerator()
+
+#examples of what kind of things we could store as params.
+
+gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
+gen.add("double_param", double_t, 0, "A double parameter",    .5, 0,   1)
+gen.add("str_param",    str_t,    0, "A string parameter",  "Hello World")
+gen.add("bool_param",   bool_t,   0, "A Boolean parameter",  True)
+
```

Comment:
This config file has test parameters and needs to be updated with the parameters we need for the bot. The other thing we can do with dynamic reconfigure is create parameter groups so we may be able to make a parameter group for each node.

Reply:
This config file also has to be set to executable using `chmod a+x mission_planner.cfg` and the config file also has to be added to the CMakeList.txt file which I have already done.

Reply:
Based on how you are implementing the mission planner, what do you think these variables will be? For example, if I wanted to tell the mission planner to just do the buoy task on repeat, how would I do that?